jenkinsstatus.py

```diff
@@ -106,20 +106,23 @@ def armada_builds(branch, debug=False, limit=20, no_color=False, no_print=False,
     NO_COLOR = no_color if not no_print else True
     build_project = "%s-build-docker" % branch
 
-    builds = all_project_builds(build_project, stderr_on_missing=stderr_on_missing)
+    session = requests.Session()
```

Comment:
I think you can do this in a `with` statement and then you don't need to call `close` implicitly